app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -37,6 +64,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/a754b9d87fd2fa98f3844adfb4608c52b022ed14/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L69"></span>Parameter item should be final.</td>
    </tr>
  </tbody>
</table>